schedule/yam/agama.yaml

```diff
@@ -8,5 +8,3 @@ schedule:
   - yam/agama/agama
   - yam/agama/agama_grub2
   - installation/first_boot
-  - yam/validate/validate_product
-  - yam/validate/validate_first_user
```

Comment:
Obviously you cannot remove this, otherwise we will not validate this in O3, if you want to use the same schedule there is a feature in openQA to exclude a coma-separate list of modules, we could try that out.

Reply:
5b383abf040e62338fbdcf8380f91a2efe572ec5